.azure-pipelines/templates/variables/globals.yml

```diff
@@ -0,0 +1,2 @@
+variables:
```

Comment:
Is there any reason this cannot go into archetype-sdk-client.yml?

Reply:
Generally agree. I'll need to test whether I can import templates into the jobs template and have it impact all jobs within that template.

Reply:
OK - what I've done is I've kept a seperate file (I think having a bunch of these settings seperately might be useful), however I now reference that template directly within the ```archetype-sdk-client``` template so the global variables don't leak up in into the ```pr.client.yml``` definitions.